spec/models/guide_spec.rb

```diff
@@ -4,6 +4,21 @@
   let!(:extra_user) { create(:user, name: 'ignatiusReilly') }
   let(:guide) { create(:guide) }
 
+  describe '#clear_progress!' do
+    let(:an_exercise) { create(:exercise) }
+    let(:another_exercise) { create(:exercise) }
+
+    before do
+      guide.exercises = [an_exercise]
+      an_exercise.submit_solution! extra_user
+      another_exercise.submit_solution! extra_user
+      guide.clear_progress!(extra_user)
+    end
+
+    it 'destroys the guides assignments for the given user' do expect(an_exercise.assignment_for(extra_user)).to be_nil end
```

Comment:
Please fix indentantation

Reply:
Seems OK to me. Am I missing something?

Reply:
As a convention, in Ruby you will use braces for one-liners, and do-end blocks for multiline lambdas